README.md

```diff
@@ -0,0 +1,19 @@
+# What is `flakeshot`?
+
+`flakeshot` is a screenshot-tool for unix-systems which runs natively on `wayland` *and* `x11`.
+
+# Motivation
+We all know some screenshot tools like [flameshot] (for X11) or [grim] (for wayland) but they
+were often only either for X11 *or* wayland. `flakeshot` should close this gap by being a screenshot
+tool which should run on X11 as good as on wayland.
+
+# Status
+`flakeshot` is still under heavy development and not considered useable.
+
+# License
+See [LICENSE.txt].
+
+[flameshot]: https://github.com/flameshot-org/flameshot
```

Comment:
Add a header like "Similar projects"

Reply:
Also why is there a LICENSE.txt under grim?

Reply:
fixed